tldr_bot.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import requests
```

Comment:
Since you are using a third-party library now, I would suggest to use a package management tool like Poetry (https://poetry.eustace.io) to capture the dependencies in Pyproject.toml.

This was the reason I steered away from using `requests`, but since you have taken it up, now might be a good time.

Reply:
This library was already used, I did not add it myself. I am not familiar with the tool, would you like to add this to the PR yourself?

Reply:
Oh it was already used ? :thinking: I must be losing my marbles then :sweat_smile: 

I am also not familiar with the tool. But that is the recommended tool for Python package management now. Since you were already working on it, I thought of suggesting it. We can add a TODO item and do it later though. Since `requests` was already used, there is no urgency for it.

Reply:
I guess opening an issue for this would be fine?

Reply:
Sure.

Reply:
Opened #7.